media-libs/mesa/mesa-9999.ebuild

```diff
@@ -383,6 +442,11 @@ multilib_src_configure() {
 		-Db_ndebug=$(usex debug false true)
 	)
 	meson_src_configure
+
+	if ! multilib_is_native_abi && use video_cards_nvk; then
+		einfo "Applying Gentoo hack for nvk - 2/2"
```

Comment:
@mattst88 check if the NVK is being apply properly

Reply:
Well for my understanding NIR dependents on the rust subprojects, it very strange it compiling one binaries as 64bits if the submodules are 32bits

Reply:
I see

```
 * Applying Gentoo hack for nvk - 1/2                                                                                                                                                                                                  
...
 * Applying Gentoo hack for nvk - 2/2                                                                                                                                                                                                  
```

And I see

```
framework /var/tmp/portage/media-libs/mesa-9999 # cat temp/rust_fix.ini 
[binaries]
rust = ['rustc', '--target=i686-unknown-linux-gnu']
```

and `--native-file /var/tmp/portage/media-libs/mesa-9999/temp/rust_fix.ini` is passed to the 32-bit configure step.

Reply:
Very strange NIR is compiling in 64 bits mode